pyproject.toml

```diff
@@ -10,7 +10,7 @@ license = {file = "LICENSE"}
 
 requires-python = ">=3.11"
 dependencies = [
-    "fastapi[all]",
+    "pandas<2.2",  # due to ray, track at: https://github.com/ray-project/ray/issues/42842
```

Comment:
This isn't necessary since it would be handled by dplutils (which is the one that brings in ray as well)

Reply:
Delete the "...." and only keep "pandas<2.2"

Reply:
no, I mean just delete the whole version spec, so line is just "pandas," . The code specific in this application don't depend on the <2.2 (probably), that comes from dplutils